torch/quantization/fx/quantize.py

```diff
@@ -237,10 +237,38 @@ def insert_observer_for_input_arg_of_observed_node(
 
 # A dictionary for querying the weight index for a given op
 WEIGHT_INDEX_DICT = {
+    torch.nn.functional.conv1d : [1],
     torch.nn.functional.conv2d : [1],
+    torch.nn.functional.conv3d : [1],
     torch.nn.functional.linear : [1],
 }
 
+def node_arg_is_weight(node: Node, arg: Any) -> bool:
+    if isinstance(node, Node) and node.op == 'call_function' and \
+            node.target in WEIGHT_INDEX_DICT:
+        for i, node_arg in enumerate(node.args):
+            if arg is node_arg and i in \
+                    WEIGHT_INDEX_DICT[node.target]:  # type: ignore
+                return True
+    return False
+
+# A dictionary for querying the weight index for a given op
+# TODO(future PR): handle linear
+BIAS_INDEX_DICT = {
+    torch.nn.functional.conv1d : [2],
+    torch.nn.functional.conv2d : [2],
+    torch.nn.functional.conv3d : [2],
+}
+
+def node_arg_is_bias(node: Node, arg: Any) -> bool:
```

Comment:
nit: shall we share the code of this function with node_arg_is_weight as well

Reply:
I don't feel too strongly.  The definition diverges in the next PR though so not sure if worth it.

Reply:
lmk if a change is needed here given the context from the next PR, I'm flexible

Reply:
sounds good, we can keep the current approach